ui/components/ui/typography/typography.js

```diff
@@ -63,6 +64,7 @@ Typography.propTypes = {
   boxProps: PropTypes.shape({
     ...Box.propTypes,
   }),
+  margin: MultipleSizes,
```

Comment:
@brad-decker also wanted to enable customizable margins for Typography components. What do you think?